src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_And_Leak.cs

```diff
@@ -82,7 +82,9 @@ public class Given_FrameworkElement_And_Leak
 		[DataRow(typeof(ToggleSwitch), 15)]
 		[DataRow(typeof(Microsoft.UI.Xaml.Controls.SwipeControl), 15)]
 		[DataRow(typeof(SplitView), 15)]
+#if !__ANDROID__ // Disabled #14341
```

Comment:
I think we should be adding a condition inside the test to exclude specific types when testing with fluent specifically, so we do not remove testing for non-fluent.